### ISSUE TITLE
elevation data?

### ISSUE THREAD

**Issue by paapu88** [HTenkanen/pyrosm#231]
Dear Developers, 

Is it possible to  show elevation data on a map using this package?
I'm not sure if openstreemap has this info, more likely opencyclemap has this.

Terveisin, Markus

**Comment by martin-kokos** [comment]
Same elements can have the `ele=` key , see documentation: https://wiki.openstreetmap.org/wiki/Key:ele

**Comment by HTenkanen** [comment]
Indeed, in theory it is possible to show elevation data based on OSM data, but it is not designed for such. Typically one would e.g. spatial join digital elevation data (DEM) with the network data nodes (if you're interested in slopes of the streets) to attach information about elevation to street networks.
